### PR TITLE
Hacktoberfest: get_height() method added to binary_search_tree.py

### PR DESCRIPTION
**Pull request by ayushkumar1610**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: data_structures/binary_tree/binary_search_tree.py [hunk 1]
```diff
@@ -107,6 +107,44 @@ def get_min(self, node=None):
                 node = node.left
         return node
 
+    def get_height(self, node) -> int:
```
Comment (NumberPiOso) [inline data_structures/binary_tree/binary_search_tree.py:110]: include typehints for node

File: data_structures/binary_tree/binary_search_tree.py [hunk 2]
```diff
@@ -107,6 +107,44 @@ def get_min(self, node=None):
...
+            left_height = self.get_height(node.left)
+            right_height = self.get_height(node.right)
+
+            # returns the largest height among both subtree
+            if left_height > right_height:
+                return left_height + 1
+            else:
+                return right_height + 1
+
     def remove(self, value):
```
Comment (NumberPiOso) [inline data_structures/binary_tree/binary_search_tree.py:148]: Include typehints

File: data_structures/binary_tree/binary_search_tree.py [hunk 3]
```diff
@@ -107,6 +107,44 @@ def get_min(self, node=None):
...
+                  9
+                 / \
+                3   10
+               / \    \
+              2   6    14
+                 / \
+                4   7
+
+        >>> bst = BinarySearchTree().insert(9, 3, 6, 2, 10, 14, 4, 7)
+        >>> bst.get_height(bst.root)
```
Comment (NumberPiOso) [inline data_structures/binary_tree/binary_search_tree.py:130]: Add doctests for floats, integers, strings, empty tree)

File: data_structures/binary_tree/binary_search_tree.py (hunk 2 above)
Comment (ayushkumar1610) [reply data_structures/binary_tree/binary_search_tree.py:148]: Could you please help me out, where should I include typehints.
Did you mean about the method remove(). If so, I can include typehint for that too.

**Comment by ayushkumar1610** [comment]
Hi @NumberPiOso 
get_height() method updated as suggested.
Thanks for reviewing.

File: data_structures/binary_tree/binary_search_tree.py (hunk 2 above)
Comment (NumberPiOso) [reply data_structures/binary_tree/binary_search_tree.py:154]: I do not know why there were some lines highligthed, I did not want to do that.

I am not a maintainer in this repository. But I guess they will ask you to include type hints in all the file.

Comment (ayushkumar1610) [reply data_structures/binary_tree/binary_search_tree.py:154]: Thanks for the suggestion, I've included all possible typehints as far as I know.

**Pull request closed without merging**